server/start.go

```diff
@@ -304,6 +305,26 @@ func startInProcess(ctx *Context, clientCtx client.Context, appCreator types.App
 
 		clientCtx := clientCtx.WithHomeDir(home).WithChainID(genDoc.ChainID)
 
+		if config.GRPC.Enable {
+			_, port, err := net.SplitHostPort(config.GRPC.Address)
+			if err != nil {
+				return err
+			}
+			grpcAddress := fmt.Sprintf("127.0.0.1:%s", port)
```

Comment:
`config.GRPC.Address` specifies both host and port, but here you force host to `127.0.0.1` in the address which you dial. If the configured address isn't reachable via that IP, will this break?

Reply:
If you cannot reach `127.0.0.1`, yes, this would not work. But this isn't the original PR, so I'm not sure why this was needed.

cc @Thunnini

Reply:
> If you cannot reach `127.0.0.1`, yes, this would not work.

I guess I'm asking about the situation where the gRPC address was set to a non-localhost IP. For example, if you have a host with a network interface that has a [public] IP of `1.2.3.4` you can bind a listener to that IP and it won't be reachable via `127.0.0.1`.